### DISCUSSION TITLE
Where are Game Data placed?

### DISCUSSION THREAD

**Discussion by samcao0416** [kwsch/PKHeX#4316] [Q&A] [answered]
Hi there!
Thank you so much for your great work! I have long been a fan of PKHeX.  Currently I'm playing a modified version of HeartGold, in which some of pkm data are changed, such as species strength and exp required at lv100. If I use the original PKHeX, some mismatching occurs. So I want to change the data in PKHeX so as to match the data in the modified ROM, and then compile my own version. I guess the game data may be placed in some txt files, but I just can't find them. Could you please tell me where these files are? Or how to modify them?
Thanks a lot!

**Comment by kwsch** [comment, marked as answer]
Information for base stats and species related permissions is stored in the PersonalInfo object. It is a data structure (not text file) that is loaded from resources.
PKHeX does not support ROM Hacks, and editing the file that stores this information is not something that regular users could/should do. You can always reset your Pokemon's stats in-game by healing at the Pokemon center or only editing box data (not party).

**Reply by samcao0416** [reply]
Got it! Thanks a lot
